docs/workflow/requirements/freebsd-requirements.md

```diff
@@ -1,81 +1,78 @@
+* Build directly on FreeBSD
 
-These instructions were validated for and on FreeBSD 12.2.
+## Docker
 
-Build using Docker on Linux
----------------------------
+Install Docker. For further instructions on installation, see [here](https://docs.docker.com/install/). You can find the official .NET images in [their Docker hub](https://hub.docker.com/_/microsoft-dotnet).
 
-This is similar to [Linux](linux-requirements.md) instructions. https://github.com/dotnet/dotnet-buildtools-prereqs-docker repro provides images
-with all needed prerequisites to build. As the example bellow may become stale, https://github.com/dotnet/versions/blob/master/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-master.json offers list of latest Docker tags.
+All the required build tools are included in the Docker images used to do the build, so no additional setup is required.
 
-```sh
-TAG=mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-freebsd-12-20210917001307-f13d79e
-docker run --rm --volume $(pwd):$(pwd) --workdir $(pwd) --env ROOTFS_DIR=/crossrootfs/x64 -ti  $TAG ./build.sh -cross -os FreeBSD
-```
+## Linux Environment
 
-Build using Toolchain Setup
----------------------------
-To build FreeBSD images, prerequisites described in [Linux](linux-requirements.md) are needed. Additionally, crossrootfs  for FreeBSD needs to be constructed.
-In order to successfully build FreeBSD crossrootfs, few more packages needs to be installed. Following example is for Ubuntu 18:
-```sh
-apt-get install -y libbz2-dev libz-dev liblzma-dev libarchive-dev libbsd-dev
-```
-With prerequisites for crossrootfs one can run:
-```sh
-./eng/common/cross/build-rootfs.sh freebsd11 $(pwd)/rootfs/freebsd
-```
-After that, FreeBSD build can be started by running
-```
-ROOTFS_DIR=$(pwd)/rootfs/freebsd ./build.sh -cross -os FreeBSD
-```
+To cross-build FreeBSD on your Linux environment, first make sure you have all the [normal Linux prerequisites](/docs/workflow/requirements/linux-requirements.md) fulfilled. Then, the _crossrootfs_ for FreeBSD has to be constructed, and this requires a few more packages to be installed:
 
+* libbz2-dev
+* liblzma-dev
+* libarchive-dev
+* libbsd-dev
 
-Building on FreeBSD
--------------------
+## FreeBSD Environment
 
-Building dotnet/runtime depends on several tools to be installed.
+These instructions assume you use FreeBSD's default binary package tool `pkg` (analog to `apt`, `apt-get`, or `yum` on Linux) to install the environment. Compiling the dependencies from source using the ports tree might work too, but is untested.
 
-Install the following packages:
+FreeBSD Prerequisites Coming Soon!
+
+Meanwhile here are the old instructions.
+
+## Old Documentation
 
-- cmake
-- icu
-- libunwind
-- lttng-ust
-- krb5
-- openssl (optional)
-- ninja (optional, enables building native code with ninja instead of make)
+These instructions were written quite a while ago, and they may or may not work today. Updated instructions coming soon.
```

Comment:
Hmm, wouldn't it be useful to keep the pre-existing package list in place before we publish the new one? I mean, there are liikely deltas but parts of this are still useful and easier to find here than elsewhere.

Reply:
OK, it looks I misread the diff, you seem to have the up-to-date package list below, I guess that means this comment can be closed.